docs/USAGE-GUIDE.md

```diff
@@ -187,33 +187,32 @@ s2n-tls uses pre-made security policies to help avoid common misconfiguration mi
+|         default         |    X   |    X   |    X   |        |    X    |    X    |      X     |      |     |     |   X   |
+|      default_tls13      |    X   |    X   |    X   |    X   |    X    |    X    |      X     |      |     |     |   X   |
+|      default_fips       |        |        |    X   |        |    X    |    X    |            |      |     |  X  |   X   |
+|        20190214         |    X   |    X   |    X   |        |    X    |    X    |            |   X  |     |  X  |   X   |
+|        20170718         |    X   |    X   |    X   |        |    X    |    X    |            |      |     |     |   X   |
+|        20170405         |    X   |    X   |    X   |        |    X    |    X    |            |   X  |     |     |   X   |
+|        20170328         |    X   |    X   |    X   |        |    X    |    X    |            |   X  |     |  X  |   X   |
+|        20170210         |    X   |    X   |    X   |        |    X    |    X    |      X     |      |     |     |   X   |
+|        20160824         |    X   |    X   |    X   |        |    X    |    X    |            |      |     |     |   X   |
+|        20160804         |    X   |    X   |    X   |        |    X    |    X    |            |   X  |     |     |   X   |
+|        20160411         |    X   |    X   |    X   |        |    X    |    X    |            |   X  |     |     |   X   |
+|        20150306         |    X   |    X   |    X   |        |    X    |    X    |            |   X  |     |     |   X   |
+|        20150214         |    X   |    X   |    X   |        |    X    |    X    |            |   X  |     |  X  |       |
+|        20150202         |    X   |    X   |    X   |        |    X    |         |            |   X  |     |  X  |       |
+|        20141001         |    X   |    X   |    X   |        |    X    |         |            |   X  |  X  |  X  |       |
+|        20190120         |    X   |    X   |    X   |        |    X    |    X    |      X     |   X  |     |     |   X   |
+|        20190121         |    X   |    X   |    X   |        |    X    |    X    |      X     |   X  |     |     |   X   |
+|        20190122         |    X   |    X   |    X   |        |    X    |    X    |      X     |   X  |     |     |   X   |
+|        20190801         |    X   |    X   |    X   |    X   |    X    |    X    |      X     |      |     |     |   X   |
+|        20190802         |    X   |    X   |    X   |    X   |    X    |    X    |      X     |      |     |     |   X   |
+|        20200207         |        |        |        |    X   |         |    X    |      X     |      |     |     |   X   |
+|        20230317         |        |        |    X   |    X   |    X    |    X    |            |      |     |     |   X   |
+|         rfc9151         |        |        |    X   |    X   |         |    X    |            |      |     |  X  |   X   |
+| CloudFront-TLS-1-2-2021 |        |        |    X   |    X   |         |    X    |      X     |      |     |     |   X   |
```

Comment:
Any reason we include a Cloudfront policy here? And should we put 20230317 at the top instead of the bottom? It's our recommended policy rn. rfc9151 should maybe also be higher.

Reply:
I think we should remove any service-specific policies. They're not really recommended to be picked up by anyone that isn't that service.